components/matches_table/commons/matches_schedule_display.lua

```diff
@@ -0,0 +1,363 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MatchesScheduleDisplay
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Abbreviation = require('Module:Abbreviation')
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local HiddenSort = require('Module:HiddenSort')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local UTC = ' <abbr data-tz="+0:00" title="Coordinated Universal Time (UTC)">UTC</abbr>'
+local WINNER_LEFT = 1
+local WINNER_RIGHT = 2
+local SCORE_STATUS = 'S'
+local DO_FLIP = true
+local NO_FLIP = false
+local LEFT_SIDE_OPPONENT = 'Left'
+local RIGHT_SIDE_OPPONENT = 'Right'
+local DEFAULT_QUERY_LIMIT = 1000
+
+-- If run in abbreviated roundnames mode
+local ABBREVIATIONS = {
+	["Upper Bracket"] = "UB",
+	["Lower Bracket"] = "LB",
+}
```

Comment:
Not needed if using Expand Headers, as it has shorted version of names

Reply:
they are for the case of manual overwrites containing them